src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -12,13 +12,15 @@
 namespace Symfony\Bridge\PhpUnit;
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
+use PHPUnit\Framework\BaseTestListener;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Collects and replays skipped tests.
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
+class SymfonyTestsListener extends BaseTestListener
```

Comment:
Yes, but we need to make it stricter. With my attempt in https://github.com/symfony/symfony/pull/21668 we might be able to make the bridge compatible with 4.8.35, 5.7 and 6.* It would be great if you could have a look, some tweaks are still needed.